### PR TITLE
[11.0][IMP] sale_secondary_unit: Set secondary uom quantity as 1.0 by default

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: sale_order_secondary_unit/models/sale_order.py [hunk 1]
```diff
@@ -63,3 +63,6 @@ def onchange_product_uom_for_secondary(self):
     @api.onchange('product_id')
     def onchange_secondary_unit_product_id(self):
```
Comment (pedrobaeza) [inline sale_order_secondary_unit/models/sale_order.py]: This should be named `_onchange_product_id_sale_order_secondary_unit`

File: sale_order_secondary_unit/models/sale_order.py [hunk 2]
```diff
@@ -63,3 +63,6 @@ def onchange_product_uom_for_secondary(self):
...
         self.secondary_uom_id = self.product_id.sale_secondary_uom_id
+        if self.secondary_uom_id:
+            self.secondary_uom_qty = 1.0
```
Comment (pedrobaeza) [inline sale_order_secondary_unit/models/sale_order.py:74]: Shouldn't this be run after quantity set and put the proper amount after conversion of the quantity to this secondary one?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
I'm not sure about the patch...

Can you put the use case and why this prevents the problem?

File: sale_order_secondary_unit/models/sale_order.py (hunk 2 above)
Comment (sergio-teruel) [reply sale_order_secondary_unit/models/sale_order.py:74]: This is only when the product is changed as a default value

**Comment by sergio-teruel** [comment]
@pedrobaeza changes done!!

**Review by rafaelbn — APPROVED** [review]
:+1: tested

@pedrobaeza please review

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**